### DISCUSSION TITLE
用AI判別Deepfake造假的影片

### DISCUSSION THREAD

**Discussion by chun1203** [1101-GS4719-C/Python-Community-GS4719-C#118] [General]
近日臺灣知名YouTuber，因用Deepfake變臉軟體製作名人不雅影片來販售牟利，引發熱議。
**Deepfake**是一種AI**影像合成技術**，透過深度學習將影片、照片中人物進行**換臉**處理。
由於近幾年，在臉書上分享Deepfake造假內容，造成各種不實的影片日益增加，所以臉書開始規範及封鎖Deepfake內容，臉書也想**以AI技術反制**這樣的亂象。但目前辨識成功率約在60%左右，距離真正可用還有段距離。

[https://www.ithome.com.tw/news/147414](url)

**Comment by Jie2222** [comment]
用AI抵制AI，無法真正的解決問題

**Comment by 109302003** [comment]
除了用ai消除在平台上的不實造假影片，我認為制定相關法規也很重要

**Comment by zixunn** [comment]
希望政府也能多宣傳正確的網路倫理觀念。

**Comment by syt-110** [comment]
感覺像是現在的亂象(製造迷因但忽視倫理方面問題)

**Comment by chupochun** [comment]
這種作法感覺治標不治本 真正要根除的是這種不正確的想法

**Comment by pekoranoinu** [comment]
樓主你的連結點不進去ㄝ。

**Reply by JunTingLin** [reply]
[用AI判別Deepfake造假的影片可行嗎？臉書：目前辨識成功率僅6成](https://www.ithome.com.tw/news/147414)

**Reply by pekoranoinu** [reply]
感恩你~\(O一O)/

**Comment by Ben6uo** [comment]
那樣的話不就可以用AI來陷害國家領導人嗎? 太可怕了
